protos/caikit_data_model_nlp.proto

```diff
@@ -27,6 +27,16 @@
 
  /*-- MESSAGES ----------------------------------------------------------------*/
 
+  message ChunkerTokenizationStreamResult {
```

Comment:
nit: Can we reference caikit proto / DM here, so that this doesn't look arbitrary and also explain why `input_start_index` etc are starting at `20`